zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Paragraph.java

```diff
@@ -105,10 +105,12 @@
   private transient Map<String, String> localProperties = new HashMap<>();
   private transient Map<String, ParagraphRuntimeInfo> runtimeInfos = new HashMap<>();
 
-  private static String  PARAGRAPH_CONFIG_RUNONSELECTIONCHANGE = "runOnSelectionChange";
+  public static String  PARAGRAPH_CONFIG_RUNONSELECTIONCHANGE = "runOnSelectionChange";
   private static boolean PARAGRAPH_CONFIG_RUNONSELECTIONCHANGE_DEFAULT = true;
-  private static String  PARAGRAPH_CONFIG_TITLE = "title";
+  public static String  PARAGRAPH_CONFIG_TITLE = "title";
   private static boolean PARAGRAPH_CONFIG_TITLE_DEFAULT = false;
+  public static String  PARAGRAPH_CONFIG_CHECK_EMTPY = "checkEmpty";
+  private static boolean PARAGRAPH_CONFIG_CHECK_EMTPY_DEFAULT = true;
```

Comment:
should this be set to the default the same as interpreter.json ?

Reply:
The default value is `checkEmpty=true`, Is consistent with the behavior of zeppelin now, If the content of the paragraph is empty, Can't execute.
Only when `checkEmpty=false`, The paragraph is empty and will be executed.

Reply:
ok.